InvenTree/InvenTree/apps.py

```diff
@@ -138,3 +146,54 @@ def update_exchange_rates(self):
                 update_exchange_rates()
             except Exception as e:
                 logger.error(f"Error updating exchange rates: {e}")
+
+    def add_user_on_startup(self):
+        """Add a user on startup"""
+        # stop if checks were already created
+        if hasattr(settings, 'USER_ADDED') and settings.USER_ADDED:
+            return
+
+        # get values
+        add_user = get_setting(
+            'INVENTREE_ADMIN_USER',
+            settings.CONFIG.get('admin_user', False)
+        )
+        add_email = get_setting(
+            'INVENTREE_ADMIN_EMAIL',
+            settings.CONFIG.get('admin_email', False)
+        )
+        add_password = get_setting(
+            'INVENTREE_ADMIN_PASSWORD',
+            settings.CONFIG.get('admin_password', False)
+        )
+
+        # check if all values are present
+        set_variables = 0
+        for tested_var in [add_user, add_email, add_password]:
+            if tested_var:
+                set_variables += 1
+
+        # no variable set -> do not try anything
+        if set_variables == 0:
+            settings.USER_ADDED = True
+            return
+
+        # not all needed variables set
+        if set_variables < 3:
+            logger.warn('Not all required settings for adding a user on startup are present:\nINVENTREE_SET_USER, INVENTREE_SET_EMAIL, INVENTREE_SET_PASSWORD')
+            settings.USER_ADDED = True
+            return
```

Comment:
Is that supposed to be `_SET_` or `_ADMIN_`? 
Few lines above you refer to them as `_ADMIN_`.

It's not breaking the functionality, just the logging